database/C/Crying Vessel/Pleasures for the Wicked/Buried Alive

```diff
@@ -0,0 +1,47 @@
+The light has come
+To take me
+(I think that I’m floating)
+Don’t know where I’m going
+Or what I’ll see
+(The darkness is growing)
+
+I can’t complain
+My time is done
+I’ve done the best that I can
+But Death has won
```

Comment:
Don't know if it's "death" or "Death", the lyric video has all-uppercase letters in it. Will confirm later with one of the band members